src/components/Blank.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react'
+import { InfoIcon } from 'react-octicons'
+
+const code = `const SmeeClient = require('smee-client')
+
+const smee = new SmeeClient({
+  source: 'https://smee.io/abc123',
```

Comment:
Could this be pre-populated with current channel url?

Reply:
yesss great idea